easybuild/easyconfigs/g/GCC/GCC-5.2.0.eb

```diff
@@ -29,7 +28,7 @@ sources = [
 
 patches = [('mpfr-%s-allpatches-20150717.patch' % mpfr_version, '../mpfr-%s' % mpfr_version)]
 
-builddependencies = [('M4', '1.4.17')]
+# builddependencies = [('M4', '1.4.17')]
```

Comment:
@shahzebsiddiqui this change should not be here; same for the other changes in this easyconfig